### DISCUSSION TITLE
細菌垂死時發出「死亡尖叫」，保護同伴逃離抗生素追殺

### DISCUSSION THREAD

**Discussion by Sherry9106** [1101-GS4719-C/Python-Community-GS4719-C#1654] [General]
當然，這些死亡尖叫是聽不見的。相反，它們是化學警報，即細菌在瀕臨死亡時傳播出來的信號，稱為「死亡信號傳遞」。

跟我之前讀過的一篇文章很類似，是關於割草的時候聞到的草味，也是草在散發出求救的訊號。

原文網址：https://kknews.cc/science/x42kyer.html

**Comment by Yanxxx7979** [comment]
這個蠻有趣的！但之前聽過忘記是youtuber還是文章有表示，像是蔬菜他們在被拔掉或是被切甚至是被吃的時候的確會散發出訊號，但其實也不一定是求救或是壞的訊號，也有可能是感受到自己的價值物盡其用？雖然我個人會覺得究竟是什麼訊號也都只是人類的猜測想像與意淫為多數罷了🤣

**Comment by THH1118** [comment]
我第一次吃香菜跟芹菜的時候他們發出的訊號讓我一輩子都不敢吃他們(雖然真正原因是他們本身的化學物質造成的www)

**Comment by WeiYen1103** [comment]
我之前看過一篇文章是在講說，在我們收割植物水果時，植物是會覺得痛的，在看完那篇文章後我覺得好酷，但這也跟很多素食主義者的想法很像，他們不吃肉不是因為宗教，而是因為他們認為在這些動物被宰殺時會覺得痛，這讓他們覺得很是慘忍，所以才選擇吃素。
